go/vt/vtgate/executor_test.go

```diff
@@ -468,6 +468,69 @@ func TestExecutorSet(t *testing.T) {
 		}
 	}
 }
+
+func TestExecutorSetMetadata(t *testing.T) {
+	*vschemaacl.AuthorizedDDLUsers = "%"
+	defer func() {
+		*vschemaacl.AuthorizedDDLUsers = ""
+	}()
+	executor, _, _, _ := createExecutorEnv()
+	session := NewSafeSession(&vtgatepb.Session{TargetString: "@master", Autocommit: true})
+
+	set := "set @@vitess_metadata.app_keyspace_v1= '1'"
+	_, err := executor.Execute(context.Background(), "TestExecute", session, set, nil)
+	if err != nil {
+		t.Errorf("%s error: %v", set, err)
+	}
+
+	show := "show vitess_metadata variables like 'app_keyspace%'"
+	result, err := executor.Execute(context.Background(), "TestExecute", session, show, nil)
+	if err != nil {
```

Comment:
Is [testify](https://godoc.org/github.com/stretchr/testify/assert) not used in Vitess? :(

Reply:
It seems to be used only in one test go/vt/vtgate/engine/ordered_aggregate_test.go

Reply:
Yeah, I just introduced it, and used it on the code I was working on at the time. You can and should definitely use it. :)

Reply:
Done. Used in the new tests